### ISSUE TITLE
How can I force sampler load order on huge workflow?

### ISSUE THREAD

**Issue by brentjohnston** [comfyanonymous/ComfyUI#3286]
I have a very large workflow that uses a ton of latent blends, SVD modifications, and like 6 samplers I'm testing. Everytime I start the workflow it decides to start with a different sampler or take different paths during the workflow, so it crashes with OOM every third time I run it or so. When it actually runs it's pretty great and I will share it when it's done.

Someone mentioned on reddit you could use "highway" nodes whatever that is, not sure if it's an extension for comfyui. 

So basically asking, how can I force which sampler starts first and also how can I force the path a latent takes if things split off multiple times with latent blends, loop back around to other samplers, etc.
